### PR TITLE
UX/DevEx: Button Components

### PR DESCRIPTION
**Pull request by ericsorenson**
Conversing buttons to use the new `Button` component.

### CODE REVIEW

File: web-client/src/ustc-ui/Button/Button.jsx [hunk 1]
```diff
@@ -20,10 +20,10 @@ export const Button = props => {
   const Element = (href && 'a') || 'button';
```
Comment (kkoskelin) [inline web-client/src/ustc-ui/Button/Button.jsx]: An unconventional ternary, sirs.

File: web-client/src/ustc-ui/Modal/ConfirmModal.jsx [hunk 2]
```diff
@@ -68,41 +69,35 @@ export class ConfirmModalComponent extends React.Component {
...
+                    icon="times-circle"
+                  />
+                </Button>
               </div>
             )}
           </div>
         </div>
         {this.props.children}
         {(!this.noConfirm || !this.noCancel) && (
-          <div className="button-box-container">
```
Comment (kkoskelin) [inline web-client/src/ustc-ui/Modal/ConfirmModal.jsx:85]: the class `.button-box-container` also imparted some styling.  Have we addressed this entirely with the new Button components?  should this class be removed from all the `scss` files?

**Review by kkoskelin — APPROVED** [review]
if my review comments are off-base, disregard and merge it. :)

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/ustc-ui/Button/Button.jsx (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/ustc-ui/Button/Button.jsx]: Updated!

File: web-client/src/ustc-ui/Modal/ConfirmModal.jsx (hunk 2 above)
Comment (ericsorenson) [reply web-client/src/ustc-ui/Modal/ConfirmModal.jsx:85]: Yes, it has been.

**Pull request merged**